root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx

```diff
@@ -215,28 +218,96 @@
     <value>Update、Deleteクエリの検索条件に指定するタイムスタンプ
 を必須にする場合は、右のチェック ボックスをオンに設定する</value>
   </data>
-  <data name="label5.Text" xml:space="preserve">
-    <value>LIKE記号：</value>
+  <data name="cbxTSIndisp.Size" type="System.Drawing.Size, System.Drawing">
+    <value>112, 17</value>
+  </data>
+  <data name="cbxTSIndisp.Text" xml:space="preserve">
+    <value>タイムスタンプ必須</value>
+  </data>
+  <data name="cbxTypedDataSet.Size" type="System.Drawing.Size, System.Drawing">
+    <value>112, 17</value>
+  </data>
+  <data name="cbxTypedDataSet.Text" xml:space="preserve">
+    <value>型付きデータセット</value>
```

Comment:
@yusukemaegawa 

Specification of this level does not  document in this project.
- This specification is self-evident, if developer read the code.
- I think that the user is possible to read the specification from this UI.

And, I thnink that it does not give big inconvenience to the user. and, users will not make recovery impossible mistake by this UI's caption. So, I will not collespond to this "pointed out". If you don't think so, please show this reason.